### ISSUE TITLE
Existing Tower automation in 'infra-ansible'

### ISSUE THREAD

**Issue by oybed** [redhat-cop/automate-tower#1]
For awareness, we have a lot of Tower Automation already in the `infra-ansible` CoP repo - both [roles](https://github.com/redhat-cop/infra-ansible/tree/master/roles/ansible/tower) and [playbooks](https://github.com/redhat-cop/infra-ansible/tree/master/playbooks/ansible/tower). 

There probably is an opportunity for collaboration to avoid dup effort.

**Comment by oybed** [comment]
@ericzolf tagging you here in case you didn't see this issue.

**Comment by ericzolf** [comment]
Yes, sorry for getting mute, it's a fair comment and I have been thinking about it. My first idea would be to rename the repo to `automate-redhat`, tower just being a sub-directory of it, and put a focus on fostering best practices of automating Red Hat own's products by:

1. linking to other repositories with automation of Red Hat products (there are many out there)
2. hosting small pieces of automation and documentation which don't justify an own repo

Does it sound like a plan?

**Comment by aoyawale** [comment]
probably better we discuss this more in a cop meeting because this is happening everywhere. We have a lot of content that is similar, that can be integrated but, we have no middle ground or who exactly makes the final decision. We are going to have a lot of disagreements on how everything is written.

**Comment by oybed** [comment]
@ericzolf the combined repo for automation of many of the Red Hat related products is what [infra-ansible](https://github.com/redhat-cop/infra-ansible) already does. Take a look at the [roles](https://github.com/redhat-cop/infra-ansible/tree/master/roles) directory for a better understand of everything that's there. Feel free to contribute there as well. 

However, I think what's more of an aim here is to "combine" multiple parallel efforts around the Tower Automation. I'll kick off an e-mail thread to discuss further, as I believe this could live under the Automation CoP - to @jlozadad 's point.

**Comment by aoyawale** [comment]
we should also take this into account https://github.com/redhat-cop/infra-ansible/issues/275

**Comment by vvaldez** [comment]
Let's definitely make this a CoP topic. I've also done Tower configuration
automation here:
https://github.com/sabre1041/managing-ocp-install-beyond/tree/master/roles/tower_config

Vinny Valdez, RHCA, RHCSS, RHCVA (110-119-187)
Sr. Principal Architect, NA Cloud Practice
Mobile: +1 (512) 718-5572
<https://www.google.com/calendar/embed?src=vvaldez%40redhat.com&ctz=America/Chicago>

**Comment by ericzolf** [comment]
And we've got a few other links from different persons since we started this discussion, internal or external, so we need definitely a strategy on how to get a "best of" of all these and join forces to maintain it.
